server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+Serialization package to transform Pulp's persistent resources from the database
+representation to a representation compatible with REST.
+
+This package supplies sub-modules, one per database collection. Each module
+supplies a number of methods to transform output for consumption by a REST
+client. The sub-modules are imported directly into this package to avoid
+namespace collisions. A developer only needs to import this package to use all
+the serialization modules.
+
+For example:
+
+from pulp.server.webservicesviews import serializeers
+bind_repr = serializers.bindings.serialize(bind)
```

Comment:
Most of these docs are generic Python things so I don't think we need to doc them. All the contents of this docstring should be deleted except for the first paragraph starting with `Serialization package`...